client/js/keybinds.js

```diff
@@ -72,6 +72,7 @@ const colorsHotkeys = {
 	u: "\x1F",
 	i: "\x1D",
 	o: "\x0F",
+	"-": "\x1e",
```

Comment:
This shortcut is used to zoom out, and I think zooming in/out is rather useful in The Lounge. What about using "s"? It's unlikely that someone ever needs to "save" the page IMO. @xPaw, thoughts?

Reply:
Agreed.

Reply:
Yes good point - updated